src/components/Footer/FooterNav/FooterNav.tsx

```diff
@@ -41,7 +43,9 @@ export const FooterNav = ({
 
   return (
     <nav className={navClasses} {...elementAttributes}>
-      {big && isExtendedNavLinks(links) && <ExtendedNav nestedLinks={links} />}
+      {big && isExtendedNavLinks(links) && (
+        <FooterExtendedNavList nestedLinks={links} />
+      )}
 
       {!isExtendedNavLinks(links) && (
         <ul className="grid-row grid-gap">
```

Comment:
I was thinking to refactor `NavList` to have props/styles for `footerNav` and then use that component here.

Reply:
Actually realized `NavList` is not as useful here since I have conditional styles on the `li` themselves. I can use the component in `FooterExtendedNavList` but not here without adding more props for list item classes to be passed in. Maybe this is where the #166 would be useful.